tests/components/MCQ.test.ts

```diff
@@ -1,25 +1,53 @@
 import { describe, test, expect } from "vitest";
 import { mount } from "@vue/test-utils";
-import { title } from "@data/question-data.json";
+import { title, options } from "@data/question-data.json";
 import MCQ from "@components/MCQ.vue";
+import { optionMount } from "./MCQUtils";
 
 describe("MCQ.vue", () => {
   test("Renders component", () => {
     const wrapper = mount(MCQ, {
       props: {
         title,
+        options,
       },
     });
     expect(wrapper.exists()).toBe(true);
     expect(wrapper.text()).toContain("MCQ Test");
   });
+
   test("Renders component with title", () => {
     const wrapper = mount(MCQ, {
       props: {
         title,
+        options,
       },
     });
     // Rather than looking at the whole wrapper, targetting specific class.
     expect(wrapper.get(".mcq-title").text()).toContain(title);
   });
+
+  test("Renders component with no options", () => {
+    const optionList = optionMount({ title, options: [] });
+    expect(optionList.length).toBe(0);
+  });
+
+  test("Renders component with one option", () => {
+    const singleOption = [{ text: "Option A", correct: true }];
+    const optionList = optionMount({ title, options: singleOption });
+    expect(optionList.length).toBe(1);
+    expect(optionList[0].text()).toBe("0: Option A");
+  });
```

Comment:
To make this test more generic: replace line39 text to a comparision with the passed options text.